lib/pages/athlete/AthletePage.dart

```diff
@@ -886,18 +886,18 @@ class _AthletePageState extends State<AthletePage> {
                             });
                           },
                           child: Icon(Icons.arrow_back,
-                              size: 50, color: Colors.white))),
+                              size: 20, color: Colors.white))),
```

Comment:
Was it really necessary to make the back arrow smaller? Change it back to what it was before.